tests/plugins/imagebase/features/manual/caption.html

```diff
@@ -0,0 +1,92 @@
+<h2>Classic editor</h2>
```

Comment:
Now that we have inline editors, please import CSS files, so that the widgets in inline editors are styled properly.

Reply:
Actually there aren't any styles that could be additionally loaded – all are already loaded. It looks differently than EI because it's a custom, test image widget:

Reply:
I mean, it's a figure - I wanted figures to be styled consistently in all editors. It just requires importing `contents.css` - see 289bd705b.